components/d2l-assessments-list-item.js

```diff
@@ -225,11 +225,9 @@ Polymer({
 			return '';
 		}
 
-		if (typeof item.icon === 'function') {
-			return item.icon(assessmentItem);
-		} else {
-			return 'd2l-tier2:' + item.icon;
-		}
+		return typeof item.icon === 'function'
+			? item.icon(assessmentItem)
+			: 'd2l-tier2' + item.icon;
```

Comment:
```suggestion
			: 'd2l-tier2:' + item.icon;
```

Why are there source changes?